README.md

```diff
@@ -36,7 +36,7 @@ Make sure you have at least 1.5 GB free disk space.
 
     ```
     curl https://sh.rustup.rs -sSf > rustup.sh
-    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(cat rust-version)
+    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(curl https://raw.githubusercontent.com/freifunk-saar/ff-node-monitor/master/rust-version)
```

Comment:
Good catch!

But this step doesn't seem right... IMO it makes more sense to just fetch the sources first.